packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -17,6 +18,12 @@ export abstract class TableColumn<
     @attr({ attribute: 'column-id' })
     public columnId?: string;
 
+    @attr({ attribute: 'action-menu-slot' })
+    public actionMenuSlot? = '';
+
+    @attr({ attribute: 'action-menu-label' })
+    public actionMenuLabel? = '';
```

Comment:
we shouldn't initialize these to empty string since that writes out to the HTML as `<element action-menu-slot="">`, instead they should default to `undefined` so just no assignment (like column-id above).